reportportal_client/service.py

```diff
@@ -214,6 +214,9 @@ def start_launch(self,
                      description=None,
                      attributes=None,
                      mode=None,
+                     uuid=None,
```

Comment:
What is uuid? UUID of the launch that is being rerun should be sent as rerunOf value. [Documentation](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md#start-launch)

Reply:
Because you should use `rerun=True + rerunOf=uuid`. UUID is auto-generated.

Reply:
You are right. "uuid" parameter is removed.